recipes-extended/resize-helper/resize-helper.bb

```diff
@@ -0,0 +1,27 @@
+SUMMARY = "Resize root filesystem to fit available disk space"
+DESCRIPTION = "Resize root filesystem to fit available disk space"
+SECTION = "admin"
+
+LICENSE = "GPLv2"
```

Comment:
I believe that `resize-helper` license is `BSD-2-Clause`

Reply:
LIC_FILES_CHKSUM = "file://resize-helper;beginline=1;endline=24;md5=output_provided_by_bitbake"